Source/React.cpp

```diff
@@ -317,7 +317,7 @@ PeleC::react_state(
 
             amrex::Real hi[NUM_SPECIES] = {0.0};
 
-#ifndef PELEC_USE_SRK
```

Comment:
While you're updating EOS stuff, how about knocking out this ifdef by adding an eos.RTY2Hi wrapper around eos.TY2Hi for Fuego (as for other eos functions) and just calling eos.RTY2Hi no matter what?

Reply:
Excellent idea. I implemented that.